misc/python/materialize/feature_benchmark/scenarios/benchmark_main.py

```diff
@@ -1833,7 +1835,7 @@ def benchmark(self) -> BenchmarkingSequence:
 class StartupLoaded(Scenario):
```

Comment:
You should bump the version of this scenario because it's no longer comparable to old results:
```
def version(self) -> ScenarioVersion:
        return ScenarioVersion.create(1, 1, 0)
```
@nrainer-materialize I think we should just make it a class variable instead of a method, but can be done independent of this PR, unless you had a good reason for this design.

Reply:
I ended up splittin out the TPC-H stuff into a new benchmark, so I think I don't need to update the version anymore, right?

Reply:
@jkosh44: That's right.